charts/pega/charts/installer/templates/pega-upgrade-environment-config.yaml

```diff
@@ -36,9 +36,49 @@ data:
 {{- if .Values.upgrade.upgradeSteps }}
   UPGRADE_STEP: {{ .Values.upgrade.upgradeSteps }}
 {{- end }}
-{{- if .Values.upgrade.pegaRESTServerURL }}
-  PEGA_REST_SERVER_URL: {{ .Values.upgrade.pegaRESTServerURL }}
-{{- end }}
+{{- $depName := "pega" }}
```

Comment:
@misterdorito minor comment -- can we move this logic to helpers.tpl?

Reply:
Moved.